pkg/el/google-fluentd-catch-all-config.spec

```diff
@@ -0,0 +1,40 @@
+Summary: Catch-all configuration files for google-fluentd on GCE.
+Name: google-fluentd-catch-all-config
+Version: 0.1
+Release: 1%{?dist}
+License: Apache
+Group: System Environment/Daemons
+URL: https://cloud.google.com/logging/
+Requires: google-fluentd
+
+%description
+A set of configuration files for google-fluentd used to ingest logs
+from the system and third-party application packages.
+
+%files
+%config /etc/google-fluentd
+
+%clean
+# don't clean up the files here; we'll do it in the toplevel Makefile
+
+# post-install/un-install scripts to handle diverting google-fluentd.conf
+%triggerin -- google-fluentd
+if [ ! -h /etc/google-fluentd/google-fluentd.conf -o ! "`readlink /etc/google-fluentd/google-fluentd.conf`" = "/etc/google-fluentd/google-fluentd.conf.google" ] ; then
+  if [ -e /etc/google-fluentd/google-fluentd.conf ] ; then
+    mv -f /etc/google-fluentd/google-fluentd.conf /etc/google-fluentd/google-fluentd.conf.orig
+  fi
+  ln -s /etc/google-fluentd/google-fluentd.conf.google /etc/google-fluentd/google-fluentd.conf
+fi
+
+%triggerun -- ntp
```

Comment:
yes, cut & paste. error - although, when I uninstalled the package it seemed to trigger this regardless.